code/modules/events/disease_outbreak.dm

```diff
@@ -165,22 +168,23 @@
 			else
 				return ADMIN_CANCEL_EVENT
 
-		chosen_max_symptoms = tgui_input_number(usr, "How many symptoms do you want your virus to have?", "A pox upon ye!", 4, 15)
+		chosen_max_symptoms = tgui_input_number(usr, "How many symptoms do you want your virus to have?", "A pox upon ye!", 4, 7, 1)
```

Comment:
label your args here so that the next person who comes around doesn't have to read the proc documentation to figure out what number means what (i had to read the proc documentation)